platform/default/mbgl/storage/offline_download.cpp

```diff
@@ -219,19 +223,26 @@ void OfflineDownload::activateDownload() {
     status = OfflineRegionStatus();
     status.downloadState = OfflineRegionDownloadState::Active;
     status.requiredResourceCount++;
-    ensureResource(Resource::style(definition.match([](auto& reg){ return reg.styleURL; }), Resource::Priority::Low),
-                   [&](Response styleResponse) {
+
+    const std::string styleURL = definition.match([&](auto& reg){ return reg.styleURL; });
+    ensureResource(Resource::style(styleURL, Resource::Priority::Low),
+                   [&, styleURL](Response styleResponse) {
         status.requiredResourceCountIsPrecise = true;
 
         style::Parser parser;
         parser.parse(*styleResponse.data);
 
+        std::string queryExtras;
+        if (static_cast<uint8_t>(options) & static_cast<uint8_t>(OfflineRegionDownloadOptions::StyleOptimizedTiles)) {
+            queryExtras = "?style=" + styleURL + "@" + parser.modified;
```

Comment:
Is `@<modified>` an official API?

Reply:
Yes - it is described in the "Request style-optimized tiles" section of the [API documentation](https://www.mapbox.com/api-documentation/#retrieve-tiles) as _timestamp_:
> The  style parameter is broken into two parts, the style ID and the style's recently edited  timestamp . The timestamp parameter comes from the style JSON's modified property, which is included with any style created with Mapbox Studio.